packages/codemod/src/utils/prop-utils.ts

```diff
@@ -104,7 +104,7 @@ export function setPropValue(
     attributePath.node.value = newValueIsTrue ? null : j.jsxExpressionContainer(j.literal(newValue.value));
   } else {
     if (newValue.type === memberExpression) {
-      const objectValue = j(`${newValue.value}`).find(j.ExpressionStatement).get().node.expression;
+      const objectValue = j(`${newValue.value}`).find(j.ExpressionStatement).get()?.node?.expression;
       attributePath.node.value = j.jsxExpressionContainer(objectValue);
```

Comment:
how does `j.jsxExpressionContainer(objectValue);` acts if `objectValue` is null or undefined? maybe early return here anyway if it is accidentally breaks